charts/incubator/funkwhale/values.yaml

```diff
@@ -1,31 +1,35 @@
-env:
-  FUNKWHALE_HOSTNAME: yourdomain.funkwhale
-  NESTED_PROXY: "0"
 image:
   pullPolicy: IfNotPresent
-  repository: tccr.io/truecharts/funkwhale
-  tag: latest@sha256:ec538fd47441c881b3f06bfffcd164b63779a805ec6d05439dbeb5e8eff52729
-persistence:
-  data:
-    enabled: true
-    mountPath: /data
-  musicpath:
-    enabled: true
-    mountPath: /music
-    readOnly: true
+  repository: funkwhale/funkwhale
+  tag: stable@sha256:b6959b3a7b0eb86248d4ccbc949865559d8ffc4b6989ce972ba7f6b7655d56bb
```

Comment:
```suggestion
  repository: tccr.io/truecharts/funkwhale
  tag: 1.2.8@sha256:e12c15ebb65102637b507ca0a610bc4a372495fd58bd8a02f2b4d024eff2badd
```

Reply:
Their `stable` tag is newer by 2 months but I mean `stable` doesn't mean release haha. Either way doesn't matter to me

Reply:
If they don't want users to use outdated images, they should tag them correctly ;)

Reply:
The honest truth lol